pkgs/development/libraries/wlroots/default.nix

```diff
@@ -1,20 +1,30 @@
 { stdenv, fetchFromGitHub, meson, ninja, pkgconfig
 , wayland, libGL, wayland-protocols, libinput, libxkbcommon, pixman
 , xcbutilwm, libX11, libcap, xcbutilimage, xcbutilerrors, mesa_noglu
+, x11Support ? true, xorg ? null, xcb ? null
 }:
 
 let pname = "wlroots";
-    version = "unstable-2018-03-16";
-in stdenv.mkDerivation rec {
+    version = "0.1";
+in stdenv.mkDerivation {
```

Comment:
```suggestion
in stdenv.mkDerivation rec {
```

Reply:
Primeos just updated the whole file, using that patch.